securedrop/tests/functional/journalist_navigation_steps.py

```diff
@@ -54,6 +54,29 @@ def _journalist_logs_in(self):
         headline = self.driver.find_element_by_css_selector('span.headline')
         assert 'Sources' in headline.text
 
+    def _journalist_visits_col(self):
+        self.driver.find_element_by_css_selector(
+            '#un-starred-source-link-1').click()
+
+    def _journalist_select_first_doc(self):
+        self.driver.find_elements_by_name('doc_names_selected')[0].click()
+
+    def _journalist_delete_one_javascript(self):
+        self.driver.find_element_by_id('delete_selected').click()
+        self._alert_wait()
+
+    def _journalist_verify_delete_one_javascript(self):
```

Comment:
changed to _journalist_verifies_deletion_of_one_submission_javascript because there will be a different code path when 

* the journalist selects all submissions instead of just one
* javascript is not active